pkg/util/azureclient/mgmt/features/subscriptionfeatureregistrations.go

```diff
@@ -0,0 +1,35 @@
+package features
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armfeatures"
```

Comment:
As part of the track2 migration  (which we haven't started but at some point will), we will move over clients one-by-one to the new track2.  I would prefer any net-new work uses the newer track2 sdk package.  

Otherwise we're going to come back to this and "reimplement" it using the newer track2 sdk.

Reply:
@bennerv thank you for this context as it will make sense to anyone looking at this code.